src/hashstore/hashstore_factory.py

```diff
@@ -28,8 +29,12 @@ def get_hashstore(self, hashstore_type):
         Raises:
             ValueError: If the given store_type is not supported.
         """
-        hashstore_type.lower()
-        if hashstore_type == "filehashstore":
-            return FileHashStore()
+
+        if hashstore_type.lower() == "filehashstore":
+            module_name = "hashstore.filehashstore.filehashstore"
+            class_name = "FileHashStore"
+            imported_module = importlib.import_module(module_name)
+            hashstore_class = getattr(imported_module, class_name)
+            return hashstore_class()
```

Comment:
is there some sort of type-checking you can do before returning the hashstore_class, to ensure it implements the HashStore interface?